waspc/packages/deploy/src/providers/fly/index.ts

```diff
@@ -23,6 +23,13 @@ class FlyCommand extends Command {
 	addLocalBuildOption(): this {
 		return this.option('--build-locally', 'build Docker containers locally instead of remotely', false);
 	}
+	addSecretsOptions(): this {
+		function collect(value: string, previous: string[]) {
+			return previous.concat([value]);
+		}
+		return this.option('--server-secret <serverSecret>', 'secret to set on the server app (of form FOO=BAR)', collect, [])
```

Comment:
So I have to do `--server-secret` for each server secret?

Looking at what Fly's CLI supports - providing a list of key value pairs, maybe we can support that like `--server-secret="ONE=1 TWO=2"`?
```
❯ flyctl secrets
Secrets are provided to applications at runtime as ENV variables. Names are
		case sensitive and stored as-is, so ensure names are appropriate for
		the application and vm environment.

Usage:
  flyctl secrets [command]

Available Commands:
  import      Set secrets as NAME=VALUE pairs from stdin
  list        List application secret names, digests and creation times
  set         Set one or more encrypted secrets for an application
  unset       Unset one or more encrypted secrets for an application
  ```

Reply:
Makes sense 👍 Docker has a similar behaviour for their `--env` option as well